src/pages/App.tsx

```diff
@@ -19,83 +19,92 @@ import Web3ReactManager from "../components/Web3ReactManager"
 import Withdraw from "./Withdraw"
 import fetchGasPrices from "../utils/updateGasPrices"
 import fetchTokenPricesUSD from "../utils/updateTokenPrices"
+import { useActiveWeb3React } from "../hooks"
 import { useDispatch } from "react-redux"
 import usePoller from "../hooks/usePoller"
 
 export default function App(): ReactElement {
+  return (
+    <Suspense fallback={null}>
```

Comment:
This looks like a big change but all I did was move the price polling stuff into `GasAndTokenPrices`, then nest everything underneath that new component.